cdap-hbase-compat-0.94/src/main/java/co/cask/cdap/data2/increment/hbase94/IncrementHandler.java

```diff
@@ -64,6 +68,13 @@
  * all the successfully committed delta values.</p>
  */
 public class IncrementHandler extends BaseRegionObserver {
+  /**
+   * Property set for {@link org.apache.hadoop.hbase.HColumnDescriptor} to configure how compaction bound is
+   * defined. Possible values are defined by {@link CompactionBound} with
+   * {@link CompactionBound#TX_UPPER_VISIBILITY_BOUND} being default.
+   */
+  public static final String PROPERTY_COMPACTION_BOUND = "increment.readless.compaction.bound";
```

Comment:
Let's make the naming of this more consistent with the property used to enable readless increments on a dataset ("dataset.table.readless.increment").  We should start namespacing any table or CF properties we set as well.  How about "cdap.readless.increment.compaction.bound"?